tensorflow_datasets/core/download/downloader.py

```diff
@@ -47,7 +47,7 @@ def get_downloader(*args: Any, **kwargs: Any) -> '_Downloader':
 def _get_filename(response: Response) -> str:
   content_disposition = response.headers.get('content-disposition', None)
   if content_disposition:
-    match = re.findall('filename="(.+?)"', content_disposition)
+    match = re.findall('[^=;\'"%]+\.[^=; \'"]+', content_disposition)
```

Comment:
There is no more `filename`. Is it expected ?

Reply:
Yup. We are searching for files with any extentions (it needs to have a dot) and searching by `filename=` ignores `filename*=`.